src/synopsys-action/synopsys-bridge.ts

```diff
@@ -126,53 +126,68 @@ export class SynopsysBridge {
       await extractZipped(downloadResponse.filePath, extractZippedFilePath)
       info('Download and configuration of Synopsys Bridge completed')
     } catch (e) {
-      const error = (e as Error).message
+      const errorObject = (e as Error).message
       await cleanupTempDir(tempDir)
-      if (error.includes('404') || error.toLowerCase().includes('invalid url')) {
+      if (errorObject.includes('404') || errorObject.toLowerCase().includes('invalid url')) {
         let os = ''
         if (process.env['RUNNER_OS']) {
           os = process.env['RUNNER_OS']
         }
         return Promise.reject(new Error('Provided Bridge url is not valid for the configured '.concat(os, ' runner')))
-      } else if (error.toLowerCase().includes('empty')) {
+      } else if (errorObject.toLowerCase().includes('empty')) {
         return Promise.reject(new Error('Provided Bridge URL cannot be empty'))
       } else {
-        return Promise.reject(new Error(error))
+        return Promise.reject(new Error(errorObject))
       }
     }
   }
 
   async prepareCommand(tempDir: string): Promise<string> {
     try {
       let formattedCommand = ''
+      const invalidParams: string[] = validateUrls()
```

Comment:
Here we are validating scan type based on passed URLs, so can we make validateUrls method name as validateScanTypes?